statsmodels/stats/proportion.py

```diff
@@ -569,6 +569,9 @@ def proportions_ztest(count, nobs, value=None, alternative='two-sided',
     TODO: add continuity correction or other improvements for small samples.
 
     '''
+    count=np.asarray(count)
+    nobs=np.asarray(nobs)
```

Comment:
space around operators like `=` (except `**`)

the main part of #2734 needs to handle scalar nobs
simple would be to add
```
if nobs.size == 1:
    nobs = nobs * np.ones(len(count))
```

(I haven't checked if vectorized would work.)